test/backend_test.in.cpp

```diff
@@ -1791,6 +1791,21 @@ NGRAPH_TEST(${BACKEND_NAME}, scalar_constant_int64)
     EXPECT_EQ(vector<int64_t>{2112}, read_vector<int64_t>(result));
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, scalar_constant_int64_1)
```

Comment:
This test should be named `scalar_constant_int64` and should replace the existing test.